closed/src/java.base/aix/native/libjncrypto/NativeCrypto_md.c

```diff
@@ -59,3 +59,10 @@ void unload_crypto_library(void *handle) {
 void * find_crypto_symbol(void *handle, const char *symname) {
     return  dlsym(handle, symname);
 }
+
+/* Find the path that the library was loaded from */
+void get_library_path(void * handle, char * library_path) {
+    if (0 != dlinfo(handle, RTLD_DI_ORIGIN, library_path)) {
+        strcpy(library_path, "Unknown");
```

Comment:
Could the message on failure be be "Unknown path" so that "Using OpenSSL version: %s (%s)\n" prints "Using OpenSSL version: 1.0.2 (Unknown path)" rather than "Using OpenSSL version:1.02. (Unknown)" and similar for the Windows and Unix versions.

Reply:
Done